[2022-12-01T16:26:06.123+0000] {taskinstance.py:1159} INFO - Dependencies not met for <TaskInstance: first_dag.pull_task manual__2022-12-01T16:26:04.815336+00:00 [queued]>, dependency 'Trigger Rule' FAILED: Task's trigger rule 'all_success' requires all upstream tasks to have succeeded, but found 1 non-success(es). upstream_tasks_state={'total': 1, 'successes': 0, 'skipped': 0, 'failed': 0, 'removed': 0, 'upstream_failed': 0, 'done': 0}, upstream_task_ids={'file_path'}
[2022-12-01T16:26:06.128+0000] {local_task_job.py:98} INFO - Task is not able to be run
